testnet/stacks-node/src/nakamoto_node/relayer.rs

```diff
@@ -99,6 +98,91 @@ impl fmt::Display for RelayerDirective {
     }
 }
 
+/// Last commitment data
+/// This represents the tenure that the last-sent block-commit committed to.
+pub struct LastCommit {
+    /// block-commit sent
+    block_commit: LeaderBlockCommitOp,
+    /// the sortition tip at the time the block-commit was sent
+    burn_tip: BlockSnapshot,
+    /// the stacks tip at the time the block-commit was sent
+    stacks_tip: StacksBlockId,
+    /// the tenure consensus hash for the tip's tenure
+    tenure_consensus_hash: ConsensusHash,
+    /// the start-block hash of the tip's tenure
+    start_block_hash: BlockHeaderHash,
+    /// What is the epoch in which this was sent?
+    epoch_id: StacksEpochId,
+    /// commit txid (to be filled in on submission)
+    txid: Txid,
+}
+
+impl LastCommit {
+    pub fn new(
+        commit: LeaderBlockCommitOp,
+        burn_tip: BlockSnapshot,
+        stacks_tip: StacksBlockId,
+        tenure_consensus_hash: ConsensusHash,
+        start_block_hash: BlockHeaderHash,
+        epoch_id: StacksEpochId,
+    ) -> Self {
+        Self {
+            block_commit: commit,
+            burn_tip,
+            stacks_tip,
+            tenure_consensus_hash,
+            start_block_hash,
+            epoch_id,
+            txid: Txid([0x00; 32]),
+        }
+    }
+
+    /// Get the commit
+    pub fn get_block_commit(&self) -> &LeaderBlockCommitOp {
+        &self.block_commit
+    }
+
+    /// What's the parent tenure's tenure-start block hash?
+    pub fn parent_tenure_id(&self) -> StacksBlockId {
+        StacksBlockId(self.block_commit.block_header_hash.clone().0)
+    }
+
+    /// What's the stacks tip at the time of commit?
+    pub fn get_stacks_tip(&self) -> &StacksBlockId {
+        &self.stacks_tip
+    }
+
+    /// What's the burn tip at the time of commit?
+    pub fn get_burn_tip(&self) -> &BlockSnapshot {
+        &self.burn_tip
+    }
+
+    /// What's the burn view consensus hash as of this last commit?
+    pub fn get_burn_view_consensus_hash(&self) -> &ConsensusHash {
+        &self.burn_tip.consensus_hash
+    }
```

Comment:
The burn view of a tenure is the block that elected the tenure (or the value of burn view in the extend tenure tx). This method is returning something else.

Reply:
I think this can just be dropped in favor of `fn get_burn_tip(&self)` above.  It's only used to see if the burnchain advanced since the last time we issued a block-commit.